tut/stream-client.md

```diff
@@ -0,0 +1,174 @@
+
+Create a `demo.html` file inside `/srv/stream-client` with the following content:
+
+```html
+<!DOCTYPE html>
+<html>
+<head>
+    <meta content="text/html;charset=utf-8" http-equiv="Content-Type">
+    <meta content="utf-8" http-equiv="encoding">
+    <title>Anbox Streaming SDK Example</title>
+</head>
+<body>
+    <script type="module">
+    import {AnboxStream, AnboxStreamGatewayConnector} from './anbox-stream-sdk.js';
+
+    const connector = new AnboxStreamGatewayConnector({
+        // Run `juju status` to get the stream gateway IP/domain name
+        // Replace 'https://gateway.url.net' with the stream gateway IP/domain name
+        url: 'https://gateway.url.net',
+        //Use the access token created earlier as 'YOUR_AUTH_TOKEN'
+        authToken: 'YOUR_AUTH_TOKEN',
+        session: {
+            // Run `amc list` to see a list of all applications
+            // Replace `com.foo.bar` with your application name
+            app: "com.foo.bar",
+        },
+        // Set the display resolution and frame rate for the Android container
+        screen: {
+            width: 1280,
+            height: 720,
+            fps: 25,
+        }
+    });
+    // The AnboxStream class takes care of the WebRTC signaling process as well as the web browser integration
+    let stream = new AnboxStream({
+        connector: connector,
+        // targetElement is the ID of the HTML element where the SDK can attach the video
+        targetElement: "anbox-stream",
+        controls: {
+            keyboard: true
+        },
+        callbacks: {
+            error: error => {
+                console.log("AnboxStream failed: ", error);
+            }
+        }
+    });
+    stream.connect();
+    </script>
+    <!--To display the video element correctly, always specify both the height and the width attributes-->
+    <div id="anbox-stream" style="width: 100vw; height: 100vh;"></div>
+</body>
+```
+If you experience any streaming issues, you can turn on debug information by adding the following option to the `AnboxStream` which is the `main` class:
+
+    experimental: {
+       debug: true,
+       }
+
+## Create and enable the stream UI service
+
+Create a `stream-ui.service` with the following content:
+```
+[Unit]
+Description=Simple Anbox Cloud Stream Client
+
+[Service]
+WorkingDirectory=/srv/stream-client
+ExecStart=/usr/bin/python3 -m http.server -b 127.0.0.1 8080
+
+[Install]
+WantedBy=multi-user.target
+```
+
+Run the following commands to install and start the service:
+```bash
+#Install the service file
+sudo cp stream-ui.service /etc/systemd/system/
+#Reload systemd
+sudo systemctl daemon-reload
+#Enable and start the service
+sudo systemctl enable --now stream-ui.service
+```
+
+## Add a middleware definition
+
+Add a middleware definition (see [Adding Basic Authentication](https://doc.traefik.io/traefik/v2.0/middlewares/basicauth/) for more details) to the [`traefik`](https://traefik.io/) configuration.
```

Comment:
Should we explain somewhere that the appliance uses traefik as component and we can reuse it for the purpose of the tutorial here?

Reply:
Added an introductory line for this.